include/taskolib/Context.h

```diff
@@ -40,15 +40,44 @@
 namespace task {
 
 /**
- * A VariableValue is a variant over several data types Lua can understand (long long, double, std::string, bool).
+ * Lua uses certain types to interface with C++.
+ *
+ * If these types are used the data comes from or shall go to the Lua engine.
+ */
+using LuaInteger = LUA_INTEGER;
+using LuaFloat = LUA_NUMBER;
+using LuaString = std::string;
+using LuaBool = bool;
+
+namespace StepVariable {
+    /**
+     * The types available to forward variables from one Step to the next.
+     */
+    using Integer = long long; ///< Storage type for integral numbers
+    using Float = double; ///< Storage type for floatingpoint number
+    using String = std::string; ///< Storage type for strings
+    using Bool = bool; ///< Storage type for booleans
+
+    /**
+     * One value holding any storage type
+     */
+    using Value = std::variant<
+        StepVariable::Integer,
+        StepVariable::Float,
+        StepVariable::String,
+        StepVariable::Bool>;
+}
```

Comment:
See commit message of commit d631f4e why I did not implement the suggested change.

That would lead to even more verbosity everywhere else.

Having `task::Integer` feels somehow ... wrong.

Reply:
(I **did** implement it at first like you suggested, but that looked so wrong/verbose/gritty when doing the test files..., esp because one can not `namespace` the `Integer` type into the current namespace.)

Reply:
> one can not `namespace` the `Integer` type into the current namespace

That is true, but one can use `using Integer = Context::Integer`. For the tests I think this is acceptable.

Reply:
... but the `VarInteger` solution is also good. :+1: